src/main/java/model/RoomSearchQuery.java

```diff
@@ -12,6 +12,8 @@
     private NumericId creator;
     private NumericId owner;
     private NumericId member;
+    private String sortOrder;
```

Comment:
Could you also add `@JsonIgnoreProperties(ignoreUnknown = true)` on top of the class? So that if another field is added in the future on the server side it won't break the SDK.